feat(spell-list.xml): add Kroderine Soul abilities; closes #16;

ondreian opened the pull request.

ondreian committed:
feat(spell-list.xml): add Kroderine Soul abilities; closes #16;

ondreian committed:
fix(spell-list.xml): escape regexp chars;

ondreian committed:
fix(spell-list.xml): fix floating point math;

amazinglarry4tw reviewed (changes requested):
There's a current issue with this implementation since it appears the start messaging is expected to be unique.  I ran some tests on my warrior, and the `Absorb Magic Recovery` will not trigger because it's superseded by `Absorb Magic`.  At least, that's the current theory.

I did a test where I changed the start messaging for the Recovery to be when the Absorb Magic buff drops and it triggered successfully... so I think if i can get the other message for when spells are absorbed, we can round this one out.

mrhoribu committed:
Update spell-list.xml

added cast proc

mrhoribu committed:
Update infomon.lic

due to infomon finding first spell with message start, start of KS absorb magic cooldown never triggers. Forcing the cooldown to trigger via infomon hackery of checking for 9655/cman absorb magic being the detected and if so, also activate the cooldown 9656.

ondreian committed:
chore: merge master -> feature/issue-16;
